tests/acceptance/conftest.py

```diff
@@ -116,3 +117,11 @@ def cleanup_workflows_tables(app):
                 obj.delete()
 
         db.session.commit()
+
+
+@pytest.fixture(autouse=True, scope='function')
+def cleanup_literatures(app):
+    recs = RecordMetadata.query.filter(RecordMetadata.json['_collections'].op('?')('Literature')).all()
```

Comment:
Can't you maybe do a straight 'delete' here with a filter or smth? I am not sure.

Reply:
mmh nope, the result of the query relies on the _collection field, which is in the json column of the table

Reply:
```python
RecordMetadata.query.filter(RecordMetadata.json['_collections'].op('?'), 'Literature')).delete(synchronize_session=False)
```

check http://docs.sqlalchemy.org/en/latest/orm/query.html?highlight=delete#sqlalchemy.orm.query.Query.delete for details.